IPCheck/parseAll.sh

```diff
@@ -1,33 +1,59 @@
 #!/bin/bash
+#*******************************************************************************
+# Copyright (c) 2021 The Eclipse Foundation
 #
-# ********************************************************************************
-# * Copyright (c) 2021, The Eclipse Foundation
-# *
-# * This program and the accompanying materials are made available under the
-# * terms of the Eclipse Public License 2.0 which is available at
-# * http://www.eclipse.org/legal/epl-2.0.
-# *
-# * SPDX-License-Identifier: EPL-2.0
-# *
-# * Contributors:
-# *   phkrief - initial implementation
-# ********************************************************************************
+# This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     phkrief - initial API and implementation
+#*******************************************************************************
 #
 #--------------------------------------------------------------
 # Runs a dependency analysis on each SmartCLIDE repo
 #--------------------------------------------------------------
 #
-./dashAnalysis.sh "Maven" "smartclide-api-gateway"
-./dashAnalysis.sh "Maven" "smartclide-broker"
-./dashAnalysis.sh "Maven" "smartclide-cicd"
-./dashAnalysis.sh "Maven" "smartclide-context"
-./dashAnalysis.sh "Maven" "smartclide-security/Theia-BackEnd" "smartclide-security"
-./dashAnalysis.sh "Maven" "smartclide-service-creation"
-./dashAnalysis.sh "Maven" "smartclide-TD-Interest"
-./dashAnalysis.sh "Maven" "smartclide-TD-Reusability-Index"
-./dashAnalysis.sh "Maven" "smartclide-TD-Principal"
-./dashAnalysis.sh "Nodes" "smartclide-ide-front-end"
-./dashAnalysis.sh "Nodes" "smartclide-perftestgen-theia"
-./dashAnalysis.sh "Yarn" "smartclide-service-creation-theia"
-./dashAnalysis.sh "Yarn" "smartclide-td-reusability-theia"
-./dashAnalysis.sh "Yarn" "smartclide-design-pattern-selection-theia"
+if [[ -z "$1" ]]; then
+	SRC_REPO="/Users/philippe/git"
+else
+	SRC_REPO=$1
+fi
+
+if [[ -z "$2" ]]; then
+	DST_REPO="/Users/philippe/git/smartclide/IPCheck/analysis"
```

Comment:
I'm wondering if we want to keep the output of the analysis in this repo? It will result in git changes whenever anyone runs the analysis.

For example, this MR contains 71 changed files but only the changes in the two `.sh` files are really relevant.

Reply:
Agree. fixed